packages/SwingSet/src/controller/controller.js

```diff
@@ -164,8 +164,7 @@ export function makeStartXSnap(bundles, { snapStore, env, spawn }) {
  *   verbose?: boolean,
  *   debugPrefix?: string,
  *   slogCallbacks?: unknown,
- *   slogFile?: string,
- *   slogSender?: (obj: any, jsonObj: string) => void,
+ *   slogSender?: import('@agoric/telemetry').SlogSender,
```

Comment:
Does this make packages/swingset dependent upon packages/telemetry? That doesn't feel like the right direction to me.. and I don't want users who only want swingset to wind up with OTEL libraries or whatnot. It doesn't need to happen now, but I'm inclined to make this type live in swingset, and have telemetry import it from there. Or, as we've discussed elsewhere for TimerService-related types, make a third package, upon which both swingset and telemetry can depend.

Reply:
This is just a type import so it doesn't introduce a runtime dependency. We can definitely move the type definition to SwingSet or something else.